transport/transport_test.go

```diff
@@ -2183,6 +2183,10 @@ func TestPingPong1MB(t *testing.T) {
 	runPingPongTest(t, 1048576)
 }
 
+// The continous ping-pong of data on a stream is meant to excercise the
+// flow-control logic to catch potential deadlocks.
+// These tests were added after a deadlock was caught by benchmarking
```

Comment:
You can leave this bit of trivia out...

Maybe say: "This is a stress-test of flow control logic"?